kernel: bump 6.1 to 6.1.73

graysky2 opened the pull request:
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.73

Patches automatically rebased.

Build system: x86/64
Build-tested: x86/64/AMD Cezanne, filogic/xiaomi_redmi-router-ax6000-ubootmod
Run-tested: x86/64/AMD Cezanne, filogic/xiaomi_redmi-router-ax6000-ubootmod

graysky2 committed:
kernel: bump 6.1 to 6.1.72

Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.72

Manually rebased:
	generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch

All other patches automatically rebased.

Build system: x86/64
Build-tested: x86/64/AMD Cezanne
Run-tested: x86/64/AMD Cezanne

Signed-off-by: John Audia <therealgraysky@proton.me>

antnyfls commented:
The patch https://github.com/openwrt/openwrt/blob/main/target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch require some manual intervention, because this changes https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.1.72&id=8b160f2fba777a27b912bcd2488e95d21f6936f0

graysky2 commented:
I am traveling and might have have a chance to look at this. If you have a fix please share it.

On Wed, Jan 10, 2024 at 7:25 PM, antnyfls ***@***.***(mailto:On Wed, Jan 10, 2024 at 7:25 PM, antnyfls <<a href=)> wrote:

> The patch https://github.com/openwrt/openwrt/blob/main/target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch require some manual intervention, because this changes https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.1.72&id=8b160f2fba777a27b912bcd2488e95d21f6936f0
>
> —
> Reply to this email directly, [view it on GitHub](https://github.com/openwrt/openwrt/pull/14383#issuecomment-1885977894), or [unsubscribe](https://github.com/notifications/unsubscribe-auth/AAECSJRJ553EQH7LS6G55NDYN4WQTAVCNFSM6AAAAABBVOJSMSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQOBVHE3TOOBZGQ).
> You are receiving this because you authored the thread.Message ID: ***@***.***>

graysky2 commented:
I am traveling and won't have a chance to look at this. If anyone has some time to spin up a fix, please, share it.

[@nbd168](https://github.com/nbd168) - since you authored [650-netfilter-add-xt_FLOWOFFLOAD-target.patch](https://github.com/openwrt/openwrt/blob/main/target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch) can you take a look?

graysky2 commented:
[@nbd168](https://github.com/nbd168) - since you authored [650-netfilter-add-xt_FLOWOFFLOAD-target.patch](https://github.com/openwrt/openwrt/blob/main/target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch) can you take a look at my rebase?

graysky2 committed:
kernel: bump 6.1 to 6.1.73

Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.1.73

Patch automatically rebased.

Build system: x86/64
Build-tested: x86/64/AMD Cezanne, filogic/xiaomi_redmi-router-ax6000-ubootmod
Run-tested: x86/64/AMD Cezanne, filogic/xiaomi_redmi-router-ax6000-ubootmod

Signed-off-by: John Audia <therealgraysky@proton.me>

graysky2 commented on target/linux/generic/hack-6.1/650-netfilter-add-xt_FLOWOFFLOAD-target.patch:
```diff
@@ -654,7 +654,7 @@ Signed-off-by: Felix Fietkau <nbd@nbd.name>
 +
 +static int nf_flow_rule_route_inet(struct net *net,
-+				   const struct flow_offload *flow,
++				   struct flow_offload *flow,
```
I am confused.  The current patch does not have the `const` on that line.  What modification are you requesting?

DragonBluep replied:
No, I approved your changes.
![image](https://github.com/openwrt/openwrt/assets/70847398/12371a5f-f296-48aa-9753-be73edb55cbf)

graysky2 replied:
OK, thanks :+1:

dick902 commented on target/linux/mediatek/patches-6.1/962-net-ethernet-mediatek-use-QDMA-instead-of-ADMAv2-on-.patch:
This need to remove as already committed in  https://github.com/openwrt/openwrt/commit/e1a6c7213fa1852d8759f4deb1243e79af49ed3e

graysky2 replied:
Done, thanks

dick902 commented on target/linux/mediatek/patches-6.1/961-net-ethernet-mediatek-split-tx-and-rx-fields-in-mtk_.patch:
This need to remove as already committed in  https://github.com/openwrt/openwrt/commit/e1a6c7213fa1852d8759f4deb1243e79af49ed3e

graysky2 replied:
Done, thanks

DragonBluep commented:
> This need to remove as already committed in [e1a6c72](https://github.com/openwrt/openwrt/commit/e1a6c7213fa1852d8759f4deb1243e79af49ed3e)

I don't understand why we have to remove these two patches. They were introduced just two days ago... https://github.com/openwrt/openwrt/commit/ede34465de2176229590028fba622d54841ab40c

dick902 commented:
> > This need to remove as already committed in [e1a6c72](https://github.com/openwrt/openwrt/commit/e1a6c7213fa1852d8759f4deb1243e79af49ed3e)
> 
> I don't understand why we have to remove these two patches. They were introduced just two days ago... [ede3446](https://github.com/openwrt/openwrt/commit/ede34465de2176229590028fba622d54841ab40c)

Those patch was duplicated as already merged in to master yesterday!  https://github.com/openwrt/openwrt/commit/e1a6c7213fa1852d8759f4deb1243e79af49ed3e

Gingernut1978 commented:
This PR, after merge, removes both these patches which seems incorrect:

961-net-ethernet-mediatek-split-tx-and-rx-fields-in-mtk_.patch
962-net-ethernet-mediatek-use-QDMA-instead-of-ADMAv2-on-.patch

graysky2 commented:
I am also confused by me removing them as I think merging this PR would delete them from master, no?

Gingernut1978 commented:
LGTM

dick902 commented:
That's work for me! THX!
